components/lists/sort/controller.tsx

```diff
@@ -29,10 +24,10 @@ export default function ListSort({
       page: "1"
     };
 
-    router.push({ pathname: router.pathname, query }, router.asPath, { shallow: false, scroll: false });
+    router.push({ pathname: `${router.pathname}`, query }, router.asPath, { shallow: false, scroll: false });
```

Comment:
we can use `router.pathname.toString()` instead, we don't need to template-string it